_blogposts/2020-02-09-release-9-0.mdx

```diff
@@ -0,0 +1,134 @@
+---
+author: hongbo
+date: "2020-02-09" 
+previewImg:
```

Comment:
I will create a release preview image template for the future to make it easier to spot in the blog overview later on

Reply:
sounds great, I can wait until tomorrow